src/js/claims-status/components/AddFilesForm.jsx

```diff
@@ -87,7 +87,7 @@ class AddFilesForm extends React.Component {
           }}>Need to mail or fax your files</a>?</p>
         </div>
         <Element name="filesList"/>
-        <div className="button-container">
+        <div>
           <ErrorableFileInput
```

Comment:
I am no longer able to test this locally, is that just on my end?

Reply:
You should still be able to do. Make sure you have the claims-mock file I sent you running, vets-api is not runnig, and there's a value in `sessionStorage.userToken` in the tab you're using.